test/test_mcmc.py

```diff
@@ -270,8 +275,8 @@ def test_mcmc_progbar():
     num_warmup, num_samples = 10, 10
 
     def model(data):
-        mean = numpyro.param('mean', 0.)
-        std = numpyro.param('std', 1., constraint=constraints.positive)
+        mean = numpyro.sample('mean', dist.Normal(0, 1).mask(False))
+        std = numpyro.sample('std', dist.LogNormal(0, 1).mask(False))
```

Comment:
@neerajprad Here is an equivalent improper prior, but with this, `init_to_prior` will work (rather than being constant at `init_value` of `numpyro.param`)